### ISSUE TITLE
https://1lib.limited can't be recognized as URL

### ISSUE THREAD

**Issue by OnceThenEver** [gdh1995/vimium-c#428]
Maybe it's due to the low-profile domain `.limited`?

**Comment by gdh1995** [comment]
Yes. Now rare words are not treated as a top-level domain, unless it has occurred in your browser history and you have opened Vomnibar once.

I'll update the logic to check bookmarks, and pass URLs like `https?://xxx\.xxx`.

**Comment by OnceThenEver** [comment]
You may
